src/zenml/zen_server/routers/users_endpoints.py

```diff
@@ -460,6 +513,15 @@ def update_myself(
         Returns:
             The updated user.
         """
+        current_user = zen_store().get_user(auth_context.user.id)
+        user.activation_token = current_user.activation_token
+        user.active = current_user.active
```

Comment:
```suggestion
        user.active = current_user.active
        user.is_admin = current_user.is_admin
```

You missed the admin flag.

Reply:
Not really, I was checking below, if the user requested a change to become an admin and is actually an admin, but it makes sense indeed not to allow to change the self admin status - other admins should do it.